package/deb_postinst

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Now that we're here: can you run shellcheck on this file and on `build_package.sh`?

Reply:
`deb_postinst` was already Shellcheck-clean, `build_package.sh` is now too.

I've silenced two errors:

```
In build_package.sh line 23:
VAULTENV_NIX_PATH=$($(nix-build --no-link -A full-build-script nix/vaultenv-static.nix) | tail -n1)
                    ^-- SC2091: Remove surrounding $() to avoid executing output.


In build_package.sh line 24:
cd -
^-- SC2103: Use a ( subshell ) to avoid having to cd back.
```

This is because the first error is intended (we want to execute here), and the second
solution is not possible because we assign to `VAULTENV_NIX_PATH`.